apiserver/src/api.rs

```diff
@@ -149,6 +153,14 @@ pub async fn run_server<T: 'static + BottlerocketNodeClient>(
             futures::future::ready(())
         });
 
+    // Set up prometheus metrics
+    let exporter = opentelemetry_prometheus::exporter().init();
+    let request_metrics = RequestMetrics::new(
+        meter("apiserver"),
```

Comment:
Do we need to set a `MeterProvider` in order for this to work?

https://docs.rs/opentelemetry/0.12.0/opentelemetry/global/index.html#usage-in-applications-1

Reply:
From my understand, NoopMeterProvider will be set as default if we don't setup anything.